pandas/core/groupby.py

```diff
@@ -2368,11 +2369,24 @@ def _get_grouper(obj, key=None, axis=0, level=None, sort=True,
     # axis of the object
     if level is not None:
         if not isinstance(group_axis, MultiIndex):
+            # allow level to be a list-like object (e.g., level=[0] or
+            # level=[0,0]) to make this case match the MultiIndex one
```

Comment:
So just to confirm: you say the following should all raise a `ValueError`: `level=[0,0]`, `level=[-1]`, `level=-1`. I do agree that the first should be an error, but I'd argue the last two should be valid --- after all, you're asking for the last level, which is meaningful also when you have just one.

@sinhrks's argument for allowing `level=[0,0]` is that it is accepted in the `MultiIndex` case.